src/main/java/core/basesyntax/FileWork.java

```diff
@@ -9,6 +17,32 @@
  */
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        Path pathToFile = Paths.get(fileName);
+        List<String> stringsFromFile = new ArrayList<>();
+        if (Files.exists(pathToFile) & Files.isReadable(pathToFile)) {
+            try {
+                Files.lines(pathToFile)
+                        .map(x -> Arrays.stream(x.split("\\W"))
+                                .map(String::toLowerCase)
+                                .filter(z -> {
+                                    if (z.startsWith("w")) {
+                                        stringsFromFile.add(z);
+                                        return true;
+                                    }
+                                    return false;
```

Comment:
it's good that you are using streams already. Im sure that you can simplify that solution and even remove arrayCreator method. 
Hint: try to use stream api methods sorted() and collect()

Reply:
done